IBX-10688: SiteAccess-aware Raptor connector configuration documented in Dev-doc (#3101)
ezsystems/developer-documentation: docs/recommendations/raptor_integration/raptor_connector.md

---
description: Step-by-step activation procedure of setting up SiteAccess-aware Raptor connector.
page_type: landing_page
---

# SiteAccess-aware Raptor connector

> The SiteAccess-aware [Raptor](https://www.raptorservices.com/) connector provides a seamless integration between [[= product_name =]] and Raptor Recommendation Engine.

Reviewer: It should be clearer that this an LTS Update. Maybe there:
```suggestion
The SiteAccess-aware [Raptor](https://www.raptorservices.com/) connector LTS Update provides a seamless integration between [[= product_name =]] and Raptor Recommendation Engine.
```

> The SiteAccess-aware [Raptor](https://www.raptorservices.com/) connector provides a seamless integration between [[= product_name =]] and Raptor Recommendation Engine.

Reviewer 2: ```suggestion
The [Raptor](https://www.raptorservices.com/) connector provides a seamless integration between [[= product_name =]] and Raptor Recommendation Engine.
```

> Its primary goal is to enable editors and managers to deliver personalized experiences across digital channels, which helps to increase conversion rates, drive sales, and improve user engagement.

Reviewer: Raptor Connector can be confusing to people knowing that there is already a Raptor connection in Ibexa CDP.

How does it work with `ibexa/cdp`? Does it need CDP to be set up? What are the scope of each? Do they interact?

```suggestion
Its primary goal is to enable editors and managers to deliver personalized experiences across digital channels, which helps to increase conversion rates, drive sales, and improve user engagement.

It must not be confused with [Ibexa CDP](cdp_guide.md) based on Raptor. CDP does TODO while Raptor Connector focus on recommandation integration TODO…
```

Author: CDP doc will be discussed after release, in separate PR. 
I'd not add this sentence here.

By combining content management capabilities with advanced recommendation features, the Connector allows teams to build and manage personalized experiences directly within a single platform.

This connector ensures a smooth and unified integration layer, enabling:

- event tracking through the tracking API
- personalized content delivery through the Recommendations API
- flexible, SiteAccess-aware configuration

This approach reduces integration complexity while providing a scalable foundation for personalization use cases across multiple sites and markets.

Follow a step-by-step procedure that allows you to activate SiteAccess-aware Raptor connector.
Activation includes configuration, adding tracking scripts and events, and using [Page Builder](page_builder_guide.md) blocks.

> [[= cards([

Reviewer 2: I'd also add relevant Help Center links here, if you find good ones

---

> description: Step-by-step activation procedure of setting up the Raptor connector.
> page_type: landing_page
> ---

Reviewer 3: ```suggestion
description: Deliver personalized experiences across digital channels with the Raptor connector.
page_type: landing_page
month_change: true
---
```

# Raptor connector

> The [Raptor](https://www.raptorservices.com/) connector provides a seamless integration between [[= product_name =]] and Raptor Recommendation Engine.

Reviewer 3: ```suggestion
The [Raptor](https://www.raptorservices.com/) connector is an [add-on](<link to add-ons on Editions page>) that provides a seamless integration between [[= product_name =]] and the recommendation engine.
```
Raptor themselves are not consistent in capitalizing the term "recommendation engine". Let's keep it lowercase.

Author: Add-ons will be covered after release.

> Its primary goal is to enable editors and managers to deliver personalized experiences across digital channels, which helps to increase conversion rates, drive sales, and improve user engagement.

Reviewer 3: ```suggestion
Its primary goal is to enable editors and managers to deliver personalized experiences across digital channels, which helps increase conversion rates, drive sales, and improve user engagement.
```

> By combining content management capabilities with advanced recommendation features, the Connector allows teams to build and manage personalized experiences directly within a single platform.

Reviewer 3: ```suggestion
By combining content management capabilities with advanced recommendation features, the connector allows teams to build and manage personalized experiences directly within a single platform.
```

> This connector ensures a smooth and unified integration layer, enabling:

Reviewer 3: ```suggestion
The connector ensures a smooth and unified integration layer, enabling:
```

- event tracking through the tracking API
- personalized content delivery through the Recommendations API
- flexible, SiteAccess-aware configuration

This approach reduces integration complexity while providing a scalable foundation for personalization use cases across multiple sites and markets.

> To configure the Raptor integration. follow a step-by-step procedure that allows you to activate the Raptor connector.

Reviewer 3: ```suggestion
To configure the integration with Raptor, follow a step-by-step procedure that allows you to activate the Raptor connector.
```
